### ISSUE TITLE
Organize channel type

### ISSUE THREAD

**Issue by josuefreire1** [filfreire/awesome-game-essays#11]
Instead of having just a "list" of channels,maybe we should categorize them by gender like:

## Dev channels:<a name="dev"></a>
- [Brackeys](https://www.youtube.com/user/Brackeys) - game development tutorials on everything from Unity and programming to game design.

## News channels:<a name="news"></a>
- [YongYea](https://www.youtube.com/user/YongYea) - game news, but also some interesting games review and analysis videos.

## Analysis channels:<a name="news"></a> 
- [Raycevick](https://www.youtube.com/channel/UC1JTQBa5QxZCpXrFSkMxmPw) - does lengthy and deep games analysis videos.

**Comment by filfreire** [comment]
Agreed 👍 
Do you want to pick up this work?

**Comment by josuefreire1** [comment]
Yes ...but Capitan I have a problem... what do I do with the channels that I don't know the category 🤔

**Comment by filfreire** [comment]
@josuefreire1 the ones you don't know the category, put under an `Uncategorized` category 😄
